### DISCUSSION TITLE
HA Os ssh

### DISCUSSION THREAD

**Discussion by Al3eRtO** [indomus/forum#2200] [GENERALE - Discussioni generiche]
Buon pomeriggio a tutti, premetto che sono alle prime armi con Ha, ho installato Ha os su rasp 4 su ssd. Non riesco a connettermi direttamente al rasp via putty, sono sicuro del nome utente e della password scelti in fase di installazione di Os. Tramite Ha via ssh accedo dal browser, con il componente aggiuntivo ma da putty niente. 
Sono riuscito durante i test ad accedere con user "Root" e password scelta in fase di installazione, ma adesso non ci riesco più. 
Cosa può essere? Sono io che mi perdo qualcosa? Grazie

**Comment by snakuzzo** [comment]
C'è più du un addon per la connessione via SSH.
Quale stai usando? Hai letto e seguito le istruzioni dell'addon?

**Reply by Al3eRtO** [reply]
Ciao, sì le ho seguite, da Ha funziona tutto, l'addon è Terminal & SSH

**Reply by snakuzzo** [reply]
Secondo me non hai seguito la parte corretta...ma io ovviamente non posso saperlo.
Perché un conto è il web terminal e un altro è la connettività in SSH.
Leggi [QUI](https://github.com/home-assistant/addons/blob/master/ssh/DOCS.md#ssh-server-connection) e segui le istruzioni
